.github/workflows/preview-web.yaml

```
@@ -0,0 +1,76 @@
+name: Preview Web
+
+on:
+  pull_request:
+    branches: ['main', feat/**, refactor/**, fix/**, chore/**]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        command: ['build']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          cache: 'pnpm'
+          cache-dependency-path: 'pnpm-lock.yaml'
+          node-version: 22
+          # node-version-file: '.nvmrc'
+
+      - name: 의존성 추가
+        run: pnpm install
+
+      - run: pnpm run ${{ matrix.command }}
+
+  vercel-preview:
+    runs-on: ubuntu-latest
+    env:
+      VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+      VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          cache: 'pnpm'
+          cache-dependency-path: 'pnpm-lock.yaml'
+          node-version: 22
+          # node-version-file: '.nvmrc'
+
+      - name: Vercel CLI 설치
+        run: pnpm install --global vercel@latest
+
+      - name: Vercel 환경 변수 가져오기
+        run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
+
+      - name: Project Artifacts 빌드하기
+        run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
+
+      - name: Vercel에 Project Artifacts 배포하기
+        id: deploy
+        run: |
+          vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }} > vercel-output.txt
+          echo "preview_url=$(cat vercel-output.txt)" >> $GITHUB_OUTPUT
+
+      - name: 현재 시간 가져오기
+        uses: josStorer/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYYMMDD-HH
+          utcOffset: '+09:00'
+
+      - name: PR 코멘트로 Preview URL 남기기
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          comment_tag: ${{github.event.number}}-vercel
+          message: |
+            🎮 React Preview 보러가기: ${{ steps.deploy.outputs.preview_url }}
```

다음과 같이 변경해도 좋을 것 같아요.
```suggestion
             Preview 보러가기: ${{ steps.deploy.outputs.preview_url }}
```

---

나중에 스토리북 Preview랑 겹칠 것 같아서 구분하려고 했습니다. 

Next나 웹으로 문구를 변경할까요? 아니면 제외해도 괜찮을까영?